arango/cursor.py

```diff
@@ -132,12 +134,18 @@ def _update(self, data: Json) -> Json:
                 self._warnings = extra["warnings"]
                 result["warnings"] = extra["warnings"]
 
+            if "plan" in extra:
+                self._plan = extra["plan"]
+                result["plan"] = extra["plan"]
```

Comment:
we're starting to blur the lines between camelCase & snake_case results...

`plan` is a dictionary, with some fields (such as `nodes` or `variables`) being list of dictionaries, whose fields are not consistent among each other. 

It becomes a slippery slope trying to convert every single field within the `nodes` & `variables` dictionaries into snake_case, so I'm opting towards just returning the results as-is.

Similar conversation here: https://github.com/arangodb/python-arango/pull/321#pullrequestreview-1872544091